src/libm/e_exp.c

```diff
@@ -98,6 +98,13 @@ P3   =  6.61375632143793436117e-05, /* 0x3F11566A, 0xAF25DE2C */
 P4   = -1.65339022054652515390e-06, /* 0xBEBBBD41, 0xC5D26BF1 */
 P5   =  4.13813679705723846039e-08; /* 0x3E663769, 0x72BEA4D0 */
 
+union {
+    Uint64 u64;
+    double d;
+} inf_union = {
+    0x7ff0000000000000
+};
```

Comment:
This is the big questionable change of this pr.
`0x7ff0000000000000` is the binary representation of a IEEE 754 64-bit double.

Reply:
Why are you concerned about this? It seems reasonable to me...

Reply:
I assumed the C standard did not say anything about floating point standard, but looks like I am wrong.
https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf
Annex F talks about floating point arithmetic.

If I wrap this into a `#ifdef  _ _STDC_IEC_559_ _ `, I think there's nothing to be concerned about anymore.

Reply:
`__STDC_IEC_559__` is not defined by mingw or MSVC.
So this is used unconditionally.
(I added a comment explaining the "magic" number)